images/runtime/php/8.2/base.bullseye.Dockerfile

```diff
@@ -0,0 +1,146 @@
+RUN sed -ri -e 's!Listen 80!Listen ${APACHE_PORT}!g' /etc/apache2/ports.conf
+# Edit Configuration to instruct Apache on how to process PHP files
+RUN echo -e '<FilesMatch "\.(?i:ph([[p]?[0-9]*|tm[l]?))$">\n SetHandler application/x-httpd-php\n</FilesMatch>' >> /etc/apache2/apache2.conf
+# Disable Apache2 server signature
+RUN echo -e 'ServerSignature Off' >> /etc/apache2/apache2.conf
+RUN echo -e 'ServerTokens Prod' >> /etc/apache2/apache2.conf
+RUN { \
+   echo '<DirectoryMatch "^/.*/\.git/">'; \
+   echo '   Order deny,allow'; \
+   echo '   Deny from all'; \
+   echo '</DirectoryMatch>'; \
+} >> /etc/apache2/apache2.conf
+
+# Install common PHP extensions
+RUN apt-get update \
+    && apt-get upgrade -y \
+    && ln -s /usr/lib/x86_64-linux-gnu/libldap.so /usr/lib/libldap.so \
+    && ln -s /usr/lib/x86_64-linux-gnu/liblber.so /usr/lib/liblber.so \
+    && ln -s /usr/include/x86_64-linux-gnu/gmp.h /usr/include/gmp.h
+
+RUN set -eux; \
+    if [[ $PHP_VERSION == 7.4.* || $PHP_VERSION == 8.0.* || $PHP_VERSION == 8.1.* || $PHP_VERSION == 8.2.* ]]; then \
+		apt-get update \
+        && apt-get upgrade -y \
+        && apt-get install -y --no-install-recommends apache2-dev \
+        && docker-php-ext-configure gd --with-freetype --with-jpeg \
+        && PHP_OPENSSL=yes docker-php-ext-configure imap --with-kerberos --with-imap-ssl ; \
+    else \
+		docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
+        && docker-php-ext-configure imap --with-kerberos --with-imap-ssl ; \
+    fi
+
+RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
+    && docker-php-ext-install gd \
+        mysqli \
+        opcache \
+        pdo \
+        pdo_mysql \
+        pdo_pgsql \
+        pgsql \
+        ldap \
+        intl \
+        gmp \
+        zip \
+        bcmath \
+        mbstring \
+        pcntl \
+        calendar \
+        exif \
+        gettext \
+        imap \
+        tidy \
+        shmop \
+        soap \
+        sockets \
+        sysvmsg \
+        sysvsem \
+        sysvshm \
+        pdo_odbc \
+# deprecated from 7.4, so should be avoided in general template for all php versions
+#       wddx \
+#       xmlrpc \
+        xsl
+
+RUN set -eux; \
+    if [[ $PHP_VERSION != 5.* ]]; then \
+        pecl install redis && docker-php-ext-enable redis; \
+    fi
+
+# https://github.com/Imagick/imagick/issues/331
+RUN pecl install imagick && docker-php-ext-enable imagick
+
+# deprecated from 5.*, so should be avoided 
+RUN set -eux; \
+    if [[ $PHP_VERSION != 5.* && $PHP_VERSION != 7.0.* ]]; then \
+        echo "pecl/mongodb requires PHP (version >= 7.1.0, version <= 7.99.99)"; \
+        pecl install mongodb && docker-php-ext-enable mongodb; \
+    fi
+
+# https://github.com/microsoft/mysqlnd_azure, Supports  7.2*, 7.3* and 7.4*
+RUN set -eux; \
+    if [[ $PHP_VERSION == 7.2.* || $PHP_VERSION == 7.3.* || $PHP_VERSION == 7.4.* ]]; then \
```

Comment:
Could you please clarify: do we even need this condition in the 8.2 dockerfile? Is there any case where this Dockerfile could be used for non 8.2 scenarios?

Reply:
Not fully sure if we need this condition here, since we dropped support for versions <= 7.4 but I saw this condition in [8.1](https://github.com/microsoft/Oryx/blob/main/images/runtime/php/8.1/base.bullseye.Dockerfile#L96) and [8.0](https://github.com/microsoft/Oryx/blob/main/images/runtime/php/8.0/base.buster.Dockerfile#L96) as well